opm/simulators/flow/LogOutputHelper.cpp

```diff
@@ -817,63 +862,62 @@ outputRegionFluidInPlace_(std::unordered_map<Inplace::Phase, Scalar> oip,
     std::ostringstream ss;
 
     ss << '\n';
-    if (reg == 0) {
-        ss << "Field total";
-    } else {
-        ss << name << " report region " << reg;
-    }
-
-    ss << " pressure dependent pore volume = "
-       << std::fixed << std::setprecision(0)
-       << cip[Inplace::Phase::DynamicPoreVolume] << ' '
-       << units.name(UnitSystem::measure::volume) << "\n\n";
```

Comment:
I thought this could be removed since this property is also reported in reservoir volumes array 

RPTSOL
 'FIPRESV' /

```
                                                     ===================================
                                                     :  RESERVOIR VOLUMES      RM3     :
 :---------:---------------:---------------:---------------:---------------:---------------:
 : REGION  :  TOTAL PORE   :  PORE VOLUME  :  PORE VOLUME  : PORE VOLUME   :  PORE VOLUME  :
 :         :   VOLUME      :  CONTAINING   :  CONTAINING   : CONTAINING    :  CONTAINING   :
 :         :               :     OIL       :    WATER      :    GAS        :  HYDRO-CARBON :
 :---------:---------------:---------------:---------------:---------------:---------------
 :   FIELD :      262665108:       87119681:      159998829:       15546597:      102666278:
 :       1 :      131333065:       37446737:       78339731:       15546597:       52993334:
 :       2 :      131332043:       49672944:       81659099:              0:       49672944:
 :       3 :              0:              0:              0:              0:              0:
 :       4 :              0:              0:              0:              0:              0:
 ===========================================================================================
```

Reply:
> I thought this could be removed since this property is also reported in reservoir volumes array

Ooops&ndash;yes, you're [absolutely right](https://github.com/OPM/opm-simulators/blob/3856421a8828413d524b263deeffc84444150a2a/opm/simulators/flow/LogOutputHelper.cpp#L908).  Thanks a lot for pointing that out!